v1/api/swagger/components/schemas/OSServiceImageDownloadHealthDetails.yaml

```diff
@@ -1,7 +1,6 @@
 type: object
```

Comment:
Not sure if renaming file is allowed, but this file name and the one that calls it are too similar & confusing.
OSServiceImageDetailedDownloadHealth.yaml
OSServiceImageDownloadHealthDetails.yaml

I'd suggest changing this file to be OSServiceImageDownloadHealthControllerDetails.yaml.

Reply:
I'll have to make some changes in quake backend once I pull the new client.